receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -36,6 +36,33 @@ query getRepoDataBySearch(
   }
 }
 
+query getRepoDataByTeam(
+    $org: String!
+    $team: String!
+    # @genqlient(pointer: true)
+    $repoCursor: String
+){
+	organization(login: $org){
```

Comment:
Let's run a pretty print on this file to fix some of this formatting. Really need to add that to a workflow but that's another task for another day.